core/middleware.py

```diff
@@ -0,0 +1,26 @@
+
+from django.http import JsonResponse
+
+'''
+Middleware to can be used to modify the way
+that requests are handled. This is how we'll
+add access control policies, validation, etc..
+'''
```

Comment:
`patterns.resource()` can accept [django style middleware](https://docs.djangoproject.com/en/3.1/topics/http/middleware/). This is helpful if we want to modify just one url route.